src/main/java/org/terasology/las/platform/FloatingPlatformRasterizer.java

```diff
@@ -30,12 +30,14 @@
 public class FloatingPlatformRasterizer implements WorldRasterizerPlugin {
 
     private BlockTheme theme;
-    private static final Block RED_DICE = CoreRegistry.get(BlockManager.class).getBlock("LightAndShadowResources:redDice");
-    private static final Block BLACK_DICE = CoreRegistry.get(BlockManager.class).getBlock("LightAndShadowResources:blackDice");
+    private Block redDice;
+    private Block blackDice;
```

Comment:
Is there any reason these were `static` before? I just made them instance variables, since they're initialized with the other instance variables, and there's only ever one instance of the rasterizer anyway.

Reply:
I don't think so. Probably was a checkstyle fix or something like that that we did without looking in too much detail.